### ISSUE TITLE
fail2ban fails to start on vanilla Debian 12

### ISSUE THREAD

**Issue by tdltdc** [geerlingguy/ansible-role-security#121]
Related to this fail2ban bug: https://github.com/fail2ban/fail2ban/issues/3292

So it's not quite this role's fault, but currently it _is_ the case that `security_fail2ban_enabled = true` fails silently (only producing a change, not an error).

A workaround (from the above mentioned bug report):
1. Add installation of `python3-systemd` to playbook:
```
- name: Install python3-systemd
  ansible.builtin.apt:
    name: python3-systemd
```
2. Set `security_fail2ban_custom_configuration_template: "jail.local"`, with `jail.local` containing at least these lines:
```
[DEFAULT]
backend = systemd
logtarget = SYSTEMD-JOURNAL
```

**Comment by stepanbaghdasaryan** [comment]
I have created a [pull request](https://github.com/geerlingguy/ansible-role-security/pull/122)  for this bug

**Comment by geerlingguy** [comment]
Thanks! I've just merged the PR. Hopefully this is fixed now—I'll test it on a Pi locally (where I noticed this bug), then tag a new release.

**Comment by geerlingguy** [comment]
This fix is included in 2.3.0.

**Comment by MaxValue** [comment]
I think `logtarget` should be `SYSOUT` if fail2ban is running as systemd service:

(mentioned in [fail2ban issue](https://github.com/fail2ban/fail2ban/issues/2227#issuecomment-421971353))
